runtime/vm/visible.c

```diff
@@ -343,6 +355,15 @@ loadAndVerifyNestHost(J9VMThread *vmThread, J9Class *clazz, UDATA options)
 		/* If a problem occurred in nest host verification then the nest host value is invalid */
 		if (J9_VISIBILITY_ALLOWED == result) {
 			clazz->nestHost = nestHost;
+		} else if (hiddenNestMate) {
+			/* 
+			 * For hidden class that is added into the nest of its host class, i.e. defined with Option.NESTMATE,
+			 * if we cannot get its nest host, its hostclass (curClazz) shuold be used as nest host instead. 
+			 */
+			clazz->nestHost = curClazz;
+			vmThread->currentException = NULL;
+			vmThread->privateFlags &= ~(UDATA)J9_PRIVATE_FLAGS_REPORT_EXCEPTION_THROW;
```

Comment:
Use the common helper to clear the exception
```suggestion
			VM_VMHelpers::clearException(vmThread);
```

Reply:
I get a compilation error if I change to use VM_VMHelpers

```
visible.c:358:38: error: expected expression before ':' token
                         VM_VMHelpers::clearException(vmThread);
                                      ^
visible.c:358:25: error: label 'VM_VMHelpers' defined but not used [-Werror=unused-label
```

Reply:
I told you to use a C++ helper in a c file.  Probably more work than it's worth to convert the file now so leave the code as it is